.github/workflows/client-tests.yaml

```diff
@@ -28,6 +28,8 @@ jobs:
 
       - name: Install dependencies
         run: cd ./client && rm -rf node_modules && yarn install --frozen-lockfile
+        run: cd ./standard_reports && rm -rf node_modules && yarn install --frozen-lockfile
+
```

Comment:
Have added a package.json file for standard_reports, and installed dependencies there (just lodash dev dependency in this case) to allow for client side tests covering standard report .test files to have their required dependencies.

I had read a number of ways of managing sharing dependencies, of which this isn't one of them! But I thought this would be the least disruptive to our structure given that it is only one dependency.